components/button/src/fixtures.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
-import { boolean } from '@storybook/addon-knobs/react';
+import { boolean, text } from '@storybook/addon-knobs/react';
 import { ButtonArrow } from '@govuk-react/icons';
 import Button from '.';
 
-export default () => <Button>Save and continue</Button>;
+export default () => (
+  <Button
+    mb="0"
+    disabled={boolean('Disabled', false)}
+    start={boolean('Start', false)}
+  >
+    {text('Children', 'Button')}
+  </Button>
+);
 
 export const ButtonStart = () => <Button start={boolean('Start', true)}>Save and continue</Button>;
```

Comment:
This could use knobs for the text, however we will refactor this later when going through everything.

Reply:
Ye the temptation to do this within this PR was very high however I think we should address it separately together over the next few working days and hopefully we can complete it on all of the components. I'll add a ticket for this so theres visibility on it.